### PR TITLE
Fix Python 3.2 compatibility

### LINKED ISSUE #161: Cannot install typing==3.5.0 for Python 3.2
The problem is with the Unicode literals in setup.py:

```
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/tmp/pip-mzzueb-build/setup.py", line 43
        author=u'Guido van Rossum, Jukka Lehtosalo, Łukasz Langa',
                                                                 ^
    SyntaxError: invalid syntax
```

I'll fix it and add a tox configuration for running tests for Python 3.2-3.4.

### PR DESCRIPTION
**Pull request by vlasovskikh**
This fixes #161.

### CODE REVIEW

**Comment by vlasovskikh** [comment]
Please review it.

File: tox.ini [hunk 1]
```diff
@@ -1,2 +1,12 @@
+[tox]
+envlist = py32, py33, py34
+
+[testenv]
+changedir = src
+commands = python -m unittest discover
+
+[testenv:py32]
+deps = mock
```
Comment (o11c) [inline tox.ini]: `mock` is not actually used, it would be better to just remove the import from `test_typing`.

File: tox.ini [hunk 2]
```diff
@@ -1,2 +1,12 @@
+[tox]
+envlist = py32, py33, py34
```
Comment (o11c) [inline tox.ini]: I know the pypi package is *currently* unneeded for python 3.5, but I think that tox should test with it.

Python 3.6 is certain to ship with a newer version of `typing`, and people running 3.5 will want to install it then in order to use the new classes.

Also, possibly comment about the 2.7 possibility (see #160).

Also enable `pypy3` (and once #160 is fixed, `pypy` and maybe `jython` - I don't have an environment installed for jython or ironpython but *ideally* they would both be supported, though they might need completely custom versions of `typing`)

File: .gitignore [hunk 3]
```diff
@@ -1,7 +1,9 @@
 MANIFEST
 build/
 dist/
+.tox/
 .idea/
 __pycache__/
+tmp/
```
Comment (o11c) [inline .gitignore]: What is `tmp/` needed for?

**Comment by gvanrossum** [comment]
Andrey, you can commit this yourself, right? Just do it.

File: tox.ini (hunk 1 above)
Comment (vlasovskikh) [inline tox.ini]: @o11c Thanks for spotting this.

File: tox.ini (hunk 2 above)
Comment (vlasovskikh) [inline tox.ini]: @o11c We can add other Python implementations eventually. Here I just wanted to fix the problem with 3.2 and establish a way to test typing against various versions. Supporting 2.7 is the next thing I am going to do.

File: .gitignore (hunk 3 above)
Comment (vlasovskikh) [inline .gitignore]: @o11c It's the directory for my temporary scripts for running tox with my local parameters and options. Not sure they worth sharing.

Comment (o11c) [inline .gitignore]: That really belongs in `.git/info/exclude` then.

**Pull request merged**